tst/functions/Coverage.Tests.ps1

```diff
@@ -465,13 +465,197 @@ InPesterModuleScope {
                 ')
             }
 
+            It 'Cobertura report must be correct' {
+                [String]$coberturaReportXml = Get-CoberturaReportXml -CommandCoverage $breakpoints -TotalMilliseconds 10000 -CoverageReport $coverageReport
+                $coberturaReportXml = $coberturaReportXml -replace 'timestamp="[0-9]*"', 'timestamp=""'
+                $coberturaReportXml = $coberturaReportXml -replace "$([System.Environment]::NewLine)", ''
+                $coberturaReportXml = $coberturaReportXml.Replace($root, 'CommonRoot').Replace($root.Replace('\', '/'), 'CommonRoot')
+                (Clear-WhiteSpace $coberturaReportXml) | Should -Be (Clear-WhiteSpace '
+                <?xml version="1.0" encoding="utf-8"?>
+                <!DOCTYPE coverage SYSTEM "http://cobertura.sourceforge.net/xml/coverage-04.dtd"[]>
+                <coverage
+                        lines-valid="16" lines-covered="14" line-rate="0.875"
+                        branches-valid="0" branches-covered="0" branch-rate="1"
+                        timestamp=""
+                        complexity="0"
+                        version="0.1">
+                    <sources><source>CommonRoot</source></sources>
+                    <packages>
+                        <package name="" line-rate="0.866666666666667" branch-rate="1">
+                            <classes>
+                                <class name="TestScript" filename="TestScript.ps1" line-rate="0.857142857142857" branch-rate="1">
+                                    <methods>
+                                        <method name="NestedFunction" hits="1" signature="()">
+                                            <lines>
+                                                <line number="5" hits="1" />
+                                                <line number="6" hits="1" />
+                                                <line number="9" hits="1" />
+                                                <line number="11" hits="1" />
+                                                <line number="12" hits="1" />
+                                                <line number="15" hits="1" />
+                                                <line number="17" hits="2" />
+                                                <line number="22" hits="0" />
+                                                <line number="25" hits="1" />
+                                                <line number="32" hits="1" />
+                                                <line number="37" hits="1" />
+                                                <line number="42" hits="0" />
+                                                <line number="46" hits="1" />
+                                                <line number="47" hits="1" />
```

Comment:
See comments above. I'm not familiar with the format, but I'd assume we'd only want lines relevant to that function here (and below). Lots of duplicated line-elements in this report atm.